packages/api/src/routes/lightclient.ts

```diff
@@ -39,15 +38,15 @@ export const routesData: RoutesData<Api> = {
   getBestUpdates: {url: "/eth/v1/lightclient/best_updates/", method: "GET"},
   getLatestUpdateFinalized: {url: "/eth/v1/lightclient/latest_update_finalized/", method: "GET"},
   getLatestUpdateNonFinalized: {url: "/eth/v1/lightclient/latest_update_nonfinalized/", method: "GET"},
-  getInitProof: {url: "/eth/v1/lightclient/init_proof/:stateId", method: "GET"},
+  getInitProof: {url: "/eth/v1/lightclient/init_proof/:epoch", method: "GET"},
```

Comment:
If we query by epoch, the consumer hopes that the state is the same as it got in the checkpoint?

Reply:
Yeah
1. start with trusted checkpoint
2. use checkpoint.root to fetch block header
3. use checkpoint.epoch to fetch init proof
4. verify init proof against header.stateRoot